TACC/Frontera-Portal
# task/FP-836: Rename Frontera-Portal repo to Core-Portal

## Overview: ##
Removes and replaces references to "Frontera Portal" with "Core Portal" or "CEP".
Also prepares links to documents and files in this repo with the repo name switch from "TACC/Frontera-Portal" to "TACC/Core-Portal".

## Related Jira tickets: ##

* [FP-836](https://jira.tacc.utexas.edu/browse/FP-836)
* [FP-839](https://jira.tacc.utexas.edu/browse/FP-839)

## Summary of Changes: ##

## Testing Steps: ##
1. Build new `core-portal:local` image: `make build`
2. Run as usual: `make start` and `npm run dev`
3. Run migrations, etc. and ensure portal is working correctly. You should see no references to "Frontera Portal" specifically

## UI Photos:
